specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2021-10-14/aiOperations_API.json

```diff
@@ -0,0 +1,134 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2021-10-14",
```

Comment:
>"2021-10-14 [](http://example.com/codeflow?start=15&length=11)

make sure this api-versions exist for "/providers/Microsoft.Insights/operations"

Reply:
I don't think you need to copy this file.

Reply:
Good point, the version must remain the same.